src/libraries/System.Net.Sockets/tests/FunctionalTests/ArgumentValidationTests.cs

```diff
@@ -800,6 +800,11 @@ public void Connect_ConnectTwice_NotSupported(int invalidatingAction)
         {
             using (Socket client = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
             {
+                // On Qemu we omit one test case due to: https://gitlab.com/qemu-project/qemu/-/issues/2410
```

Comment:
After addressing https://github.com/dotnet/runtime/pull/104094/files#r1666697753, this can be replaced as:

```diff
-    // On Qemu we omit one test case due to: https://gitlab.com/qemu-project/qemu/-/issues/2410
+    // On Qemu we omit one test case due to: [ActiveIssue(....)]
```

So it becomes searchable for us.